docs/docs/using-fragments.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Using Fragments
+title: Using GraphQL Fragments
```

Comment:
We should probably update the filename as well. Here are some instructions: https://www.gatsbyjs.org/contributing/docs-contributions/#docs-renaming-instructions

Reply:
You're right!

Reply:
Could you give me some feedback on this? Here's my attempt to follow those instructions :) --> https://github.com/gatsbyjs/gatsby/pull/18762/commits/411fe9869b78a828e5b2f402918a2c6dd0549887